bindgen-integration/build.rs

```diff
@@ -123,13 +123,9 @@ impl ParseCallbacks for MacroCallback {
     // Test the "custom derives" capability by adding `PartialEq` to the `Test` struct.
     fn add_derives(&self, name: &str) -> Vec<String> {
         if name == "Test" {
-            vec![
-                "PartialEq".into(),
-            ]
+            vec!["PartialEq".into()]
```

Comment:
These changes seem drive-by?

Reply:
I think they happened because cargo didn't consider `bindgen-integration` a proper crate before so `rustfmt` never ran over it.